Tests iOS/MyList/MyListTests.swift

```diff
@@ -243,7 +243,7 @@ class MyListTests: XCTestCase {
         app
             .webReaderView
             .staticText(matching: "Hello, world")
-            .wait()
+            .wait(timeout: 2.0)
```

Comment:
@jacobsimeon This UI test was failing on local and bitrise both. I verified with checking out the develop locally and it was still failing. I had to bump up the timeout to 2.0

Reply:
Looks good!